server/util/priority_queue/priority_queue.go

```diff
@@ -98,23 +98,23 @@ func (pq *PriorityQueue[V]) Push(v V, priority int) {
 	})
```

Comment:
I think the `noValueGetter` field could be removed now? otherwise LGTM

Reply:
Done.